src/Config.php

```diff
@@ -59,6 +59,26 @@ public static function getStoreIdsByType(string $type): array {
     return $ids;
   }
 
+  public static function getStoreIdsByName(string $name): array {
+    $ids = [];
+    foreach (Config::get() as $id => $store_config) {
+      if ($store_config['name'] === $name) {
+        $ids[] = $id;
+      }
+    }
+    return $ids;
```

Comment:
Not sure if this will work, but if you just want to extract IDs from the config, you can probably use `array_filter` to extract the config values and then return the keys

```suggestion
    return array_keys(array_filter(Config::get(), fn($item) => $item['name'] === $name));
```